python/py-filelock/Portfile

```diff
@@ -17,18 +17,29 @@ long_description    This package contains a single module, which implements \
                     a platform independent file lock in Python, which \
                     provides a simple way of inter-process communication:
 
-homepage            https://filelock.readthedocs.io/en/latest/
+homepage            https://py-filelock.readthedocs.io/en/latest/api.html
 
-checksums           rmd160  f9852c8432196648df2648529d21c14df1d121a1 \
-                    sha256  34a9f35f95c441e7b38209775d6e0337f9a3759f3565f6c5798f19618527c76f \
-                    size    205796
+checksums           rmd160  9d2213165c6dfc44a0652124fb8a74ee2bd3ac5b \
+                    sha256  93d512b32a23baf4cac44ffd72ccf70732aeff7b8050fcaf6d3ec406d954baf4 \
+                    size    206359
 
 # keep version for PY27 and PY34, these are (indirect) dependencies of py-virtualenv
 python.versions     27 34 35 36 37 38 39 310
 
 if {${name} ne ${subport}} {
     depends_build-append \
-                    port:py${python.version}-setuptools_scm
+                    port:py${python.version}-setuptools_scm \
+                    port:py${python.version}-toml
```

Comment:
the `toml` dependency is only needed for Python 3.5 and below as the newer `setupools_scm` ports depend on `tomli` already